src/oge/load_data.py

```diff
@@ -221,7 +221,9 @@ def update_epa_to_eia_map(cems_df: pd.DataFrame):
     )
 
     # update the eia plant ids
-    cems_df["plant_id_eia"].update(cems_df["plant_id_eia_manual"])
+    cems_df.update(
+        pd.DataFrame({"plant_id_eia": cems_df["plant_id_eia_manual"].to_list()})
```

Comment:
Does this approach preserve the NA values in the column being sent to list so that we are only overwriting non-NA values? I just want to make sure we are not accidentally replacing data we do not intend. (This applies to all instances of this solution throughout the PR)

Reply:
I believe it does as the below example shows:
```python
>>> df = pd.DataFrame({'A': [1.0, 2.0, 3.0],
...                    'B': [400.0, 500.0, np.NaN]})
>>> df
     A      B
0  1.0  400.0
1  2.0  500.0
2  3.0    NaN
>>> df.update({"A": df["B"].tolist()})
>>> df
       A      B
0  400.0  400.0
1  500.0  500.0
2    3.0    NaN
>>> 

```